scss/core/_forms.scss

```diff
@@ -211,7 +211,7 @@ input[type="radio"],
 .#{$fd-namespace}-radio {
     border-radius: 50%;
     &::after {
-        $check-size_: fd-space(2);
+        $check-size_: 10px;
```

Comment:
Although the change itself is good, I think the PR (and commit) should be prefixed with `fix:` and not `chore:`.  Chores are meant to be for infrastructure changes within the repo that don't alter the distribution package.  This clearly does change the distribution package.

Also, I think you need to fix the link to the issue that it will close -- it currently points to another merged PR.